charts/airbyte/values.yaml

```diff
@@ -0,0 +1,380 @@
+  ## @param webapp.ingress.hosts Ingress Hosts configuration
+  ## @param webapp.ingress.annotations [object] Ingress annotations done as key:value pairs
+  ## @param webapp.ingress.hosts The list of hostnames to be covered with this ingress record.
+  ## @param webapp.ingress.tls [array] Custom ingress TLS configuration
+  ingress:
+    enabled: false
+    className: ""
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    hosts:
+    - host: chart-example.local
+      paths:
+      - path: /
+        pathType: ImplementationSpecific
+    tls: []
+    # - secretName: chart-example-tls
+    #   hosts:
+    #   - chart-example.local
+
+
+## @section Scheduler Parameters
+
+scheduler:
+  ## @param scheduler.replicaCount Number of scheduler replicas
+  replicaCount: 1
+
+  ## @param scheduler.image.repository The repository to use for the airbyte scheduler image.
+  ## @param scheduler.image.pullPolicy the pull policy to use for the airbyte scheduler image
+  ## @param scheduler.image.tag The airbyte scheduler image tag. Defaults to the chart's AppVersion
+  image:
+    repository: airbyte/scheduler
+    pullPolicy: IfNotPresent
+    tag: 0.29.13-alpha
+
+  ## @param scheduler.podAnnotations [object] Add extra annotations to the scheduler pod
+  ##
+  podAnnotations: {}
+
+  ## Scheduler resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param scheduler.resources.limits [object] The resources limits for the scheduler container
+  ## @param scheduler.resources.requests [object] The requested resources for the scheduler container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 200m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    memory: 256Mi
+    ##    cpu: 250m
+    requests: {}
+
+  ## @param scheduler.nodeSelector [object] Node labels for pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## @param scheduler.tolerations [array] Tolerations for scheduler pod assignment.
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+
+## @section Pod Sweeper parameters
+
+podSweeper:
+  ## @param podSweeper.image.repository The image repository to use for the pod sweeper
+  ## @param podSweeper.image.pullPolicy The pull policy for the pod sweeper image
+  ## @param podSweeper.image.tag The pod sweeper image tag to use
+  image:
+    repository: bitnami/kubectl
```

Comment:
@davinchia we should pick a specific version for this instead of using `latest`. We do this in the Kustomize version as well.